src/NServiceBus.Core/Pipeline/Incoming/SatelliteTransportReceiveToPhysicalMessageProcessingConnector.cs

```diff
@@ -0,0 +1,15 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Pipeline;
+
+    class SatelliteTransportReceiveToPhysicalMessageProcessingConnector : StageConnector<ITransportReceiveContext, IIncomingPhysicalMessageContext>
```

Comment:
Since we want the satellite pipeline completely separate from the "main" pipeline should we just give it a raw "SatelliteReceiveContext" instead to avoid sharing anything with the main pipeline?

This would also remove "physical" from the lingo of the satellite pipeline since there isn't any "logicalmessage" part to that pipeline?